File: app/src/main/java/com/snaptiongame/app/data/models/Rank.java
package com.snaptiongame.app.data.models;

import com.google.gson.annotations.SerializedName;

/**
 * @author Tyler Wong
 */

public class Rank {
    @SerializedName(ID)
    public int id;
    @SerializedName(TITLE)
    public String title;

    public static final String ID = "id";
    public static final String TITLE = "title";
}

File: app/src/main/java/com/snaptiongame/app/data/models/Rank.kt
package com.snaptiongame.app.data.models

import com.google.gson.annotations.SerializedName

/**
 * @author Tyler Wong
 */

class Rank {
    @SerializedName(ID)
    var id: Int = 0
    @SerializedName(TITLE)
    var title: String? = null

    companion object {
        const val ID = "id"
        const val TITLE = "title"
    }
}
